cpg_workflows/stages/gatk_sv/gatk_sv_single_sample.py

```diff
@@ -0,0 +1,249 @@
+            'WGD_matrix': 'WGD_scoring_matrix_output.bed.gz',
+            'WGD_scores': 'WGD_scores.txt.gz',
+            'bincov_matrix': 'RD.txt.gz',
+            'bincov_matrix_index': 'RD.txt.gz.tbi',
+            'bincov_median': 'medianCov.transposed.bed',
+            'qc_table': 'evidence_qc_table.tsv',
+        }
+        for caller in SV_CALLERS:
+            for k in ['low', 'high']:
+                fname_by_key[f'{caller}_qc_{k}'] = f'{caller}_QC.outlier.{k}'
+
+        return {key: self.prefix / fname for key, fname in fname_by_key.items()}
+
+    def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput:
+        d = inputs.as_dict_by_target(GatherSampleEvidence)
+        sids = cohort.get_sample_ids()
+
+        input_dict: dict[str, Any] = {
+            'batch': cohort.name,
+            'samples': sids,
+            'run_vcf_qc': True,
+            'counts': [str(d[sid]['coverage_counts']) for sid in sids],
+        }
+        for caller in SV_CALLERS:
+            input_dict[f'{caller}_vcfs'] = [
+                str(d[sid][f'{caller}_vcf']) for sid in sids
+            ]
+
+        input_dict |= get_images(
+            [
+                'sv_base_mini_docker',
+                'sv_base_docker',
+                'sv_pipeline_docker',
+            ]
+        )
+
+        input_dict |= get_references(['genome_file', 'wgd_scoring_mask'])
+
+        expected_d = self.expected_outputs(cohort)
+        jobs = add_gatk_sv_jobs(
+            batch=get_batch(),
+            dataset=cohort.analysis_dataset,
+            wfl_name=self.name,
+            input_dict=input_dict,
+            expected_out_dict=expected_d,
+        )
+        return self.make_outputs(cohort, data=expected_d, jobs=jobs)
+
+
+@stage(required_stages=EvidenceQC, analysis_type='sv', analysis_keys=['batch_json'])
+class CreateSampleBatches(CohortStage):
+    """
+    uses the values generated in EvidenceQC
+    splits the samples into batches based on median coverage,
+    PCR +/- status, and Sex
+    """
+
+    def expected_outputs(self, cohort: Cohort) -> dict[str, Path]:
+        return {'batch_json': self.prefix / 'pcr_{pcr_status}_batches.json'}
+
+    def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
+        expected = self.expected_outputs(cohort)
+
+        # PCR +/- logic is only relevant to exomes
+        if get_config()['workflow'].get('sequencing_type') != 'genome':
+            sample_types = {'exome': [sam.id for sam in cohort.get_samples()]}
+        # within exomes, divide into PCR- and all other samples
+        else:
+            sample_types = {'positive': [], 'negative': []}
+            for sample in cohort.get_samples():
+                if sample.meta.get('pcr_status', 'unknown') == 'negative':
+                    sample_types['negative'].append(sample.id)
+                else:
+                    sample_types['positive'].append(sample.id)
```

Comment:
@EddieLF can you check this logic in terms of keywords, location in meta/database schema, see if it gels with what you are thinking?